views/Game.js

```diff
@@ -0,0 +1,40 @@
+import { View, Text, TouchableOpacity, StyleSheet } from "react-native";
+import { GameEngine } from "react-native-game-engine";
+import Physics from "../utils/physics";
+import { useGame } from "../hooks/useGame";
+import { StartButton, ScoreLabel } from "../components";
+
+
+export function Game() {
+  const {
+    entities,
+    running,
+    currentPoints,
+    handleUpdateCurrentPoints,
+    handleSetGameEngine,
+    startGame,
+    systems,
+  } = useGame();
```

Comment:
I am importing some of my states like `running`, `currentPoints`, etc here. 

Since I am using the hook here, it is the same thing as calling useState on this page, but now we just islolate our logic. 

Unlike context, I can't just call `useGame` in any component I want access to the value in, so I call it in my `view`, and then pass it down as a `prop` to my child component

This keeps my components atomic, relying on data from a view to populate and make them interactive.